### DISCUSSION TITLE
Consulta de instalacion - GlobalLeaks

### DISCUSSION THREAD

**Discussion by rodrigzchela** [globaleaks/globaleaks-whistleblowing-software#4028] [General]
Buenas tardes, 

Quisiera saber si es posible instalar globalleaks a traves de la pagina o plataforma o si hay alguna alternativa de instalacion que no sea en un servidor/equipo local.

Muchas gracias por las dudas que puedan evacuar.

**Comment by evilaliv3** [comment]
Buenas tardes @rodrigzchela ,

El siguiente mensaje está traducido al español automáticamente, espero que sea comprensible.

Para instalar globalaks necesita un servidor Linux Ubuntu o Debian privado o virtual. Puede encontrar las especificaciones de requisitos y los detalles de instalación aquí.

Si le ayuda, tenemos una empresa social que puede proporcionar el software como servicio SaaS. Para información puede escribirnos a info@globaleaks.org
